quipucords/scanner/satellite/utils.py

```diff
@@ -131,7 +136,11 @@ def execute_request(scan_task, url, org_id=None, host_id=None,
             ssl_verify = source_options.ssl_cert_verify
         host, port, user, password = get_connect_data(scan_task)
     url = construct_url(url, host, port, org_id, host_id)
+    connect_timeout = int(settings.QPC_SSH_CONNECT_TIMEOUT[:-1])
```

Comment:
you could add a comment here to make the slicing a little more clear.

Reply:
Done